libc/spec/stdc.td

```diff
@@ -898,6 +898,15 @@ def StdC : StandardSpec<"stdc"> {
       ]
   >;
 
+  HeaderSpec StdFix = HeaderSpec<
+      "stdfix.h",
+      [],  // macros
```

Comment:
Do all of the newly added macros from libc/include/llvm-libc-macros/stdfix-macros.h need to go in this list?

Reply:
Those are for macros generated inside `%%public_api` block, similar to: https://github.com/llvm/llvm-project/blob/main/libc/config/linux/api.td#L30

Since we put all of their generic definitions in `stdfix-macros.h` to be used by internal implementations, they are not needed here.  Otherwise, it will create redefinitions of those macros.